R/download_mun_list_by_year.R

```diff
@@ -0,0 +1,26 @@
+load_bfs_mun_list <- function(date_or_year) {
+  # FIXME: filtering by canton not easily possible, since the canton is not part of the downloaded table
```

Comment:
Cantons are at `level == 1` in this data?

Reply:
yes, thanks, this works